src/index.js

```diff
@@ -12,171 +12,179 @@ const dirScanner = require('./dirScanner')
 const ScanStudy = require('./ScanStudy')
 const HashDataWriter = require('./HashDataWriter')
 const JSONReader = require('./JSONReader')
-const {getArg,hasArg, getRemainingArgs, showHelp} = require('./../src/args');
+const { getArg, hasArg, getRemainingArgs, showHelp } = require('./../src/args');
 const path = require('path');
 const homedir = require('os').homedir();
+
 const dicomwebDefaultDir = '~/dicomweb';
 const Stats = require('./stats');
 const VideoWriter = require('./VideoWriter')
+const OverallStats = new Stats('OverallStats', 'Overall statistics');
+const StudyStats = new Stats('StudyStats', 'Study Generation', OverallStats);
+const handleHomeRelative = dirName => dirName[0] == '~' ? (path.join(homedir, dirName.substring(1))) : dirName;
+
+class StaticWado {
```

Comment:
we might have conflicts with your PR and mine. the first in, second should fix conflicts

Reply:
Always an issue with multiple developers.